apisix/init.lua

```diff
@@ -185,6 +185,9 @@ function _M.http_ssl_phase()
         end
         ngx_exit(-1)
```

Comment:
Should we also clear the `ngx.ctx` here?

Reply:
If the SSL handshake fails, there will be no more HTTP requests.

Reply:
@membphis 
You are right.